src/woo_search/search_index/index.py

```diff
@@ -16,6 +16,16 @@
 from .typing import IndexName, InformatieCategorieType, PublisherType
 
 
+class Attachment(InnerDoc):
+    content: M[str] = mapped_field(
+        Text(
+            fields={"keyword": Keyword()},
```

Comment:
this is expected to be a large blob of text - why are you adding keyword subfields to this?

Reply:
The Process Attachment Ingest pipeline by default sets up a keyword field besides the normal text one. I kept the behavior to make sure the same implementation remains. We don't use it anywhere in the project so we can get rid of it, but I didn't know for certain if we wanted to alter it's default functionality this drastically or not.

Reply:
I think it's safe to omit it :)